### PR TITLE
Add initial privacy_policy.md file 

### LINKED ISSUE #287: Privacy Policy
We need this since we are logging trip planner and eventually geocoder requests.

### PR DESCRIPTION
**Pull request by jmfield2**
Also add a link to github from the left menu - Send Feedback

Fix #287

### CODE REVIEW

**Comment by jmfield2** [comment]
@barbeau

File: privacy_policy.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+
+USF Maps Project Privacy Policy
+
+We value your privacy, so we want to be explicit about what kind of personal information is involved when you interact with USF Maps.
+
+This page is for the USF Maps project as a whole. The project as such doesn’t collect any personally identifying information from users.  However, instances of USF Maps that serve a particular region may save log and other information, some of which is personally identifiable. Each instance should have its own privacy policy, linked from its home page.
```
Comment (barbeau) [inline privacy_policy.md]: We can delete this paragraph for now - once we figure out multiple campuses/white-labeling we can revisit multiple instances language.

File: privacy_policy.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+USF Maps Apps
+
+Usage of the trip planner function logs information about the requested start and endpoints, date/time, requested modes, and user IP address.  As a result of the nature of the application, it may also associate the users geolocation at the time of the request.  We realize some of this data can be sensitive, and do not plan to use it in a way that unnecessarily reveals such information to third-parties.  We expect to only make use of the information for troubleshooting problems, and conducting research on the aggregate, anonymized information contained therein.
+
+We use Google Analytics so aggregate, anonymous data about how often different app features are used can be collected.  For example, we could tell that the most used feature of USF Maps apps is the trip planner, and only 1/3 of users regularly use the maps screen (these statistics are made up for example purposes).  This data helps us better understand where to focus future improvements.  We do not collect data on individual usage of the app.  For example, we don’t know how many times User X viewed their person bookmarks, or names of those bookmarks.
```
Comment (barbeau) [inline privacy_policy.md]: Can change last sentence to:

> For example, we don’t know how many times User X used the trip planning function vs. the layers function.

File: privacy_policy.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+In the “Help” section of the apps, the app allows users to send a feedback email to the regional email address – the user can also add additional recipients to this email.  The following information is automatically added to the body of the email to help us troubleshoot issues that users encounter, with the message “The below information helps us troubleshooting problems:". The user can erase this information from the email before it is sent if the user desires not to share this information with the regional USF Maps administrators or the app maintainers:
```
Comment (barbeau) [inline privacy_policy.md]: "regional email address" can just change to "email address".

File: privacy_policy.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+    USF Maps App version – So we know what known bugs do and do not exist in your currently-installed version of the app
```
Comment (barbeau) [inline privacy_policy.md]: You can remove "USF Maps App version", but should add "Browser platform and version".

File: privacy_policy.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    Device make and model (e.g., iPhone 7.2, Samsung Galaxy S5) – Helps us troubleshoot device-specific problems
+    OS version (e.g., Android 6.0, iOS 9.3.1) – Helps us troubleshoot operating system version-specific problems
+
+The USF Maps app currently uses the respective APIs to show the user their real-time location and location of bus stops. These Maps APIs have fairly broad data collection abilities, and this data goes back to the respective platform vendor (Google, Apple, Microsoft). The USF Maps project doesn’t receive this data however.
```
Comment (barbeau) [inline privacy_policy.md]: "this data goes back to the respective platform vendor (Google, Apple, Microsoft)" 

...can be changed to:

"some data may be collected by the respective platform vendor (Mapbox, MapQuest)."

File: privacy_policy.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+We want to be completely transparent about what data is collected in the apps, and if you’re interested in digging through the source code, it’s available on GitHub:
+
+https://github.com/CUTR-at-USF/usf-mobullity/mobullityrebase
+
+Other Apps Using USF Maps Services
```
Comment (barbeau) [inline privacy_policy.md]: We can remove this paragraph - again applies to regional design.

File: privacy_policy.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+The USF Maps servers in different regions often provide data for other transit applications in addition to the official USF Maps ones. Those apps should include their own privacy policies if they collect personal data.
+
+Beta Test Program for Apps
```
Comment (barbeau) [inline privacy_policy.md]: Let's remove beta testing section as well.

File: privacy_policy.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+Project members will periodically release updated versions of the apps, or create new apps. We will often recruit volunteers to help beta test these apps before releasing them.
```
Comment (barbeau) [inline privacy_policy.md]: Can remove this paragraph too.

File: privacy_policy.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+Beta Test Program for Apps
+
+Project members will periodically release updated versions of the apps, or create new apps. We will often recruit volunteers to help beta test these apps before releasing them. 
+
+Specific Research Projects
+
+USF Maps is also a research project, in addition to being a service. We may therefore invite some USF Maps users to participate in specific research projects in which we gather more detailed information. However, participation in any such study will be voluntary and we will always ask
+your permission before enrolling you. We will follow all the standard Human Subjects protocols for such a study, including requesting your informed consent. Any other research projects will only use non-personally identifiable data already collected by the project as a whole or a particular installation, in accordance with this policy and the policy for the particular installation.
+
+Sample Regional Privacy Policy
```
Comment (barbeau) [inline privacy_policy.md]: Cam remove "Sample Regional Privacy policy" section.

**Comment by barbeau** [comment]
Thanks @jmfield2!  Looks good, some comments inline.

**Pull request merged**